### PR TITLE
feat(modular-styles): add remaining imports, update source import

### PR DESCRIPTION
**Pull request by SimonFinney**
## Affected issues

- Resolves partially #44

## Proposed changes

- Update remaining components
- Fix missing imports
- Update source imports for compilation, Storybook and theme addon
- Remove fixed style overrides
- Clean up

### CODE REVIEW

File: .storybook/addons/addon-theme/register.js [hunk 1]
```diff
@@ -12,8 +12,7 @@ import { action, getStoredTheme, namespace, themes, title } from '.';
 
 import { Form, Select, SelectItem } from '../../../src';
 
-// TODO: `modular-styles` - Revert.
-// import '../../../src/index.scss';
+import './index.scss';
```
Comment (j1mie) [inline .storybook/addons/addon-theme/register.js]: 👍

**Comment by SimonFinney** [comment]
Thanks @j1mie - After revisiting and fixing merge conflicts with `dev`, this is ready to go back into the feature branch so we can migrate it over into `dev`

**Comment by SimonFinney** [comment]
@j1mie Good catch! If it's alright with you, I'd like to go with the latter and merge this into the `modular-styles` feature branch - those issues appear to be a by-product of the merge from `dev` as they exist there too.

**Pull request merged**